web/src/utils/factory-management/syncState.ts

```diff
@@ -32,24 +32,28 @@ export const calculateSyncState = (factory: Factory) => {
     return
   }
 
```

Comment:
Just thinking here as well, if the factory is **not** in sync, is there any point in checking this as we only want to check if the factory has dropped out of sync.

If the user marks a factory as "in sync" at amount 1, increases it to 2 then reduces it back down to 1, the factory doesn't go back in sync. Therefore we only really care if the factory is currently regarded as in sync.